app/assets/javascripts/proxy_status/dhcp.js

```diff
@@ -0,0 +1,63 @@
+function activateDhcpTables() {
```

Comment:
I suggest using the foreman namespace. see application.js:29
I'd put all this in foreman.controls.dhcp or something similar.